code/game/objects/lighting/roguelighting.dm

```diff
@@ -620,127 +620,90 @@
 		else if(istype(attachment, /obj/item/reagent_containers/glass/bucket/pot))
 			var/obj/item/reagent_containers/glass/bucket/pot = attachment
 			if(istype(W, /obj/item/reagent_containers/food/snacks/grown/oat))
-				if(!pot.reagents.has_reagent(/datum/reagent/water, 51))
+				if(!pot.reagents.has_reagent(/datum/reagent/water, 49))
 					to_chat(user, "<span class='notice'>Not enough water.</span>")
 					return TRUE
 				if(pot.reagents.chem_temp < 374)
 					to_chat(user, "<span class='warning'>[pot] isn't boiling!</span>")
 					return
-				if(do_after(user,2 SECONDS, target = src))
+				if(do_after(user,1 SECONDS, target = src))
 					user.visible_message("<span class='info'>[user] places [W] into the pot.</span>")
 					qdel(W)
 					playsound(src.loc, 'sound/items/Fish_out.ogg', 20, TRUE)
-					pot.reagents.remove_reagent(/datum/reagent/water, 51)
+					pot.reagents.remove_reagent(/datum/reagent/water, 50)
 					sleep(300)
 					playsound(src, "bubbles", 30, TRUE)
 					pot.reagents.add_reagent(/datum/reagent/consumable/soup/oatmeal, 50)
```

Comment:
will this mess up if you take the pot off the stove or put other stuff in?

Reply:
originally i added this code
addtimer(CALLBACK(pot, TYPE_PROC_REF(/obj/item/reagent_containers/glass/pot, makeSoup), nutrimentamount), rand(1 MINUTES, 3 MINUTES))

Reply:
> will this mess up if you take the pot off the stove or put other stuff in?

It'll work just fine unless you for some reason fill up the pot with other liquids to the point where there's no more space for the soup to be created.

So no, it won't mess up unless you intentionally make it mess up.

Reply:
still i think its a little weird that you can pick up the pot and it remains empty and then a couple minutes later soup randomly appears

Reply:
maybe like an inprogress soup reagent or something?

Reply:
> maybe like an inprogress soup reagent or something?

That's a good call, along with making the pot uninteractible until it's done cooking.

Reply:
yea i think uninteractible is a good solution, you wouldnt need incomplete reagent in that situation since you wouldnt even be able to put the pot off

only problem is there would need to be a system for multiple soups  being cooked at once